nncf/common/accuracy_aware_training/runner.py

```diff
@@ -262,6 +313,79 @@ def __init__(self, accuracy_aware_params: Dict[str, object], verbose=True,
         self.training_epoch_count = 0
         self.cumulative_epoch_count = 0
         self.best_val_metric_value = 0
+        self.current_val_metric_value = 0
+
+        self.optimizer = None
+        self.lr_scheduler = None
+
+        self.base_lr_reduction_factor_during_search = 1.0
+        self.lr_updates_needed = lr_updates_needed
+        self.load_checkpoint_fn = None
+        self.early_stopping_fn = None
+        self.update_learning_rate_fn = None
+
+        self._train_epoch_fn = None
```

Comment:
It looks inconsistent when some attributes are prefixed with `_` and some are not.

Reply:
Renamed all *fns to start with `_`